### PR TITLE
implement mysql

### PR DESCRIPTION
**Pull request by yalh76**
## Problem

- *congressus_ynh require a mysql galette*

## Solution

- *implement mysql*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by yalh76** [comment]
!testme

File: conf/config.inc.php.mysql [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+*/
+
+/* choose your database engine, values : pgsql|mysql */
+define("TYPE_DB", "mysql"); 
+/* hostname for the database */
+define("HOST_DB", "localhost"); 
+/* tcp port for the database */
+define("PORT_DB", "3306"); 
+/* the username for the database */
+define("USER_DB", "__DB_NAME__");
```
Comment (ericgaspar) [inline conf/config.inc.php.mysql:17]:
```suggestion
define("USER_DB", "__DB_USER__");
```

**Review by ericgaspar — CHANGES_REQUESTED** [review]

File: conf/config.inc.php.mysql (hunk 1 above)
Comment (yalh76) [reply conf/config.inc.php.mysql]: Fixed ^^

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
!testme

**Pull request merged**